platform/admin/ui/src/main/webapp/js/application.js

```diff
@@ -62,6 +64,7 @@ define([
         mainRegion: 'main',
         appHeader: '#appHeader',
         modalRegion: '#modalRegion',
+        sessionTimeoutModalRegion: '#sessionTimeoutModalRegion',
```

Comment:
I dont think this is being used anywhere - the modal appends to modalRegion on wreqr.vent.trigger('showModal', new SessionTimeoutModal)

Reply:
See above. I think we can remove this.